### ISSUE TITLE
Twitter Strategy

### ISSUE THREAD

**Issue by bencodezen** [vuejs/events#42]
**Is your feature request related to a problem? Please describe.**
Social media strategy can be difficult to execute properly for those that don't spend a lot of time in it.

**Describe the solution you'd like**
Provide a strategy for how to tweet about events and such in a low effort but high ROI way.

**Comment by tessamero** [comment]
@bencodezen , do you mind if I do a write up for this? If I do, I will write my Hootsuite strategy, and how I categorize and automate all of my social media content for the conference I organize, I can also include how I do meetup social media (this part isn't automated as it's more excited when it's "live tweeted". 

Let me know :)

**Comment by bencodezen** [comment]
@tessamero Not at all. I would love to hear about your strategy! This one is all yours!

**Comment by tessamero** [comment]
I will make this as a todo but not priority item =)

**Comment by bencodezen** [comment]
@tessamero You got it. No worries!

**Comment by tessamero** [comment]
Would it make sense to add that to this section here? https://events.vuejs.org/resources/#_5-spread-the-word
